src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -33,18 +32,18 @@
               </div>
               <div class="col-md-2 col-sm-2 col-xs-3 no-padding user-avatar">
                 <img *ngIf="item.relationalData?.assignees?.length"
-                    src="{{item.relationalData.assignees[0]?.attributes.imageURL + '&s=20'}}"
-                    onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'" />
+                     src="{{item.relationalData.assignees[0]?.attributes.imageURL + '&s=20'}}"
+                     onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'" />
                 <span *ngIf="!item.relationalData?.assignees?.length" class="fa fa-user not-assigned-user-icon"></span>
               </div>
             </aside>
-
-           </div>
+            <div class="clear"></div>
```

Comment:
It is for clearing the float So, that next element is not affected by previous element float property.

You can check all about float [here](https://css-tricks.com/all-about-floats/).